components/01-atoms/lists/_list-item.twig

```diff
@@ -0,0 +1,15 @@
+{#
+ # Available variables:
+ # - list__item__content - the content of the list_item (typically text)
+ #
+ # Available blocks:
+ # - list__item__content - used to replace the content of the list_item with something other than text
+ #   for example: to insert the image and/or link components
+ #}
+{% set list__item__base_class = list__item__base_class|default('item') %}
+
+<li {{ bem(list__item__base_class, list__item__modifiers, list__item__blockname) }}>
```

Comment:
While I can't think of a good example of where/why one would do this I think we should refactor this to use the `add_attributes(li__attributes)` method instead. This allows the `<li>` to be easily extended in the future (if needed) as well as consistency with the already established components.

Reply:
I'm still not convinced this is necessary. The `bem` function exists, and is a simpler implementation that `add_attributes`. So, in my opinion, if you're only setting the `class` attribute, you should use the simpler method. Refactoring isn't hard, and switching to `add_attributes` later, if a need arrises would be my preferred plan of action. But it's not a big deal here, so I went ahead and switched them both.